demo/rn/metro.config.js

```diff
@@ -0,0 +1,48 @@
+const path = require("path");
```

Comment:
This file is the magic that allows for importing the development files into the demo app, even though they live two levels above the demo app.